src/components/heading/Heading.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Link } from 'react-router-dom';
 import './Heading.css';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 const heading = () => (
   <>
     <header className="heading">
       <div className="nav">
         <Link to="/">
-          <h2 className="store">BookStore</h2>
+          <h2 className="store">BookStore CMS</h2>
         </Link>
         <Link to="/Books" className="books">
```

Comment:
- [SUGGESTION] Let this one also link to home. Currently, when clicked on, we get an empty page. Why? Because the path `"/Books"`, is not defined in the routes in `App.js`, therefore the app does not know which component to display when this link is clicked. Consider 👇🏼:

  ```suggestion
          <Link to="/" className="books">
  ```